src/lang/bg-BG.json

```diff
@@ -86,7 +86,7 @@
   "app_pvgna": "Find a Guide",
   "app_pvgna_alt": "Find a Dota 2 Guide on Pvgna",
   "app_rivalry": "Bet on Pro Matches",
-  "app_rivalry_team": "Bet on %s Matches",
+  "app_rivalry_team": "Bet on {1} Matches",
```

Comment:
shouldn't we start at 0 for consistency with c#?

Reply:
I can fix that.  Some reason I got the impression C# started at 1 🙃

Reply:
here are the docs for reference:
https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=netframework-4.7.2